### DISCUSSION TITLE
Issue Creating Two or More Instances in ScadaLTS v2.7.4.1

### DISCUSSION THREAD

**Discussion by LucasFaleirosMoreira** [SCADA-LTS/Scada-LTS#2987] [Q&A]
Hello, I’m trying to create two instances on the same ScadaLTS (e.g., localhost:8080/ScadaBR and localhost:8080/CBR), but I’m encountering some difficulties. I’m able to create the instances and access them via the URL in the web browser; however, one of the instances has an issue where I cannot modify any DataSource (clicking edit/copy shows the message "System Exception"), while the other instance works normally.

To create the instances, I followed these steps: I created a new directory called CBR and copied the ScadaBR files (var/lib/tomcat9/webapps/ScadaBR) to CBR, set the permissions for the user tomcat
on the new directory, created a database for CBR, and then modified the context.xml file (var/lib/tomcat9/webapps/CBR/META-INF/context.xml) in the section for username="database user" password="database password" and url="jdbc:mysql://localhost:3306/cbr". I also run the GRANT ALL PRIVILEGES command on the created database in MySQL.

Does anyone know how to resolve this System Exception error and use more than one instance in a single ScadaLTS v2.7.4.1? If possible.

**Comment by Limraj** [comment]
Hi @LucasFaleirosMoreira,

The problem is probably that one session is being shared by two instances. You can check this by logging in on two browsers, then we can be sure that we are operating in different sessions. Unfortunately, we store some edited objects in the user from a given session, i.e. only one Data Source can be edited at a time in a given session, we have solved this problem for Graphical Views. (https://github.com/SCADA-LTS/Scada-LTS/issues/2275)
We also recommend using the latest version of the application, or the stable version 2.6.18.

Regards,
Kamil Jarmusik
